plugins/stabilities.nix

```diff
@@ -0,0 +1,16 @@
+# Capacitor API (scope == floxpkgs)
+{lib, ...}:
+# User API
+# -- no user input required --
+
+# Plugin API (scope == Using flake)
+{context, capacitate,...}:
+
+{
+  __reflect.finalFlake.config.stabilities = {
```

Comment:
should document the asssumptiions. eg: "there must be a root defined which has a nixpkgs with these three stabilities.

or even better, something like
```
context.root.inputs.nixpkgs.stable or builtins.throw "The root flake must be defined. It must have a nixpkgs input. That input must have stable,staging,unstable attributes".
```

Reply:
This PR should state that it was mainly for experimental usage
If we add this plugin into production it sure had more checks applied.